### PR TITLE
Add docker-compose, remove indexer, move to node4

### PR DESCRIPTION
**Pull request by nledez**
@nono, @kosssi any comments?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -35,16 +35,9 @@ RUN pip install \
...
- && wget -q -O - http://nodejs.org/dist/v0.10.40/node-v0.10.40.tar.gz | tar xz \
- && cd node-v0.10.40 \
- && ./configure \
- && CXX="g++ -Wno-unused-local-typedefs" make \
- && CXX="g++ -Wno-unused-local-typedefs" make install \
- && cd .. \
- && rm -rf /tmp/node-v* \
- && npm install -g npm@latest-2
+# Install NodeJS 4.2.X LTS
+RUN curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
```
Comment (nono) [inline Dockerfile]: @nledez is `sudo` necessary here?

**Comment by nono** [comment]
Looks good to me

**Comment by kosssi** [comment]
You can update documentation to talk your docker-compose.yml

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cozy_data:
```
Comment (kosssi) [inline docker-compose.yml:2]: The norme of yml is always start your file with "---"

```
---
iam: yml
```

**Comment by kosssi** [comment]
else for me is :+1:

File: Dockerfile (hunk 1 above)
Comment (nledez) [inline Dockerfile]: Oups

**Comment by nono** [comment]
Thanks @nledez!

**Pull request merged**